base/VulkanInitializers.hpp

```diff
@@ -90,13 +90,14 @@ namespace vks
 			return imageMemoryBarrier;
 		}
 
-		/** @brief Initialize a buffer memory barrier with no image transfer ownership */
+		/** @brief Initialize a buffer memory barrier with no buffer transfer ownership */
 		inline VkBufferMemoryBarrier bufferMemoryBarrier()
 		{
 			VkBufferMemoryBarrier bufferMemoryBarrier {};
 			bufferMemoryBarrier.sType = VK_STRUCTURE_TYPE_BUFFER_MEMORY_BARRIER;
 			bufferMemoryBarrier.srcQueueFamilyIndex = VK_QUEUE_FAMILY_IGNORED;
 			bufferMemoryBarrier.dstQueueFamilyIndex = VK_QUEUE_FAMILY_IGNORED;
+			bufferMemoryBarrier.size = VK_WHOLE_SIZE;
```

Comment:
`VkBufferMemoryBarrier::size` being zero is results in an invalid barrier, so everywhere you're using this helper, you must be assigning to the size, meaning this will always be overwritten if you're being explicit about the size (and is therefore innocuous as a change) but it means everywhere you're explicitly setting it to `VK_WHOLE_SIZE` could eliminate that line.

Reply:
This is only the initializer. All places that use it set the buffer size afterwards.

Reply:
I can revert this change and make sure all my new barriers explicitly set it then.